pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -73,8 +91,13 @@
 # Instrument test attributes
 
 _test_dates = {'': {'vtec': dt.datetime(2017, 11, 19),
-                    'site': dt.datetime(2001, 1, 1)}}
-
+                    'site': dt.datetime(2001, 1, 1),
+                    'los': dt.datetime(2023, 1, 1)}}
+_test_load_opt = {'': {'los': [{'los_method': 'site', 'los_value': 'zzon',
+                                'gnss_network': 'glonass'},
+                               {'los_method': 'time',
+                                'los_value': dt.datetime(2023, 1, 1)}]}}
+_test_download = {'': {'los': False}}  # Download is too large to test
```

Comment:
```suggestion
_test_download_ci = {'': {'los': False}}  # Download is too large to test
```
This will allow you to test locally, but skip it on the CI. The other flag forces a test to make sure that download has a warning since it assumes that downloads are not available.

If you just want to skip downloads locally too, use `_password_req = {'': {'los': True}}`.

https://pysat.readthedocs.io/en/latest/new_instrument.html#testing-support